dnsvizor.opam

```diff
@@ -0,0 +1,16 @@
+opam-version: "2.0"
+maintainer: "team@robur.coop"
+authors: ["team@robur.coop"]
+homepage: "https://github.com/roburio/dnsvizor"
+doc: "https://roburio.github.io/dnsvizor/doc"
+dev-repo: "git+https://github.com/roburio/dnsvizor.git"
+bug-reports: "https://github.com/roburio/dnsvizor/issues"
+license: "public domain"
+
```

Comment:
what's the point of this file?

Reply:
(a) get travis-opam to work (b) install the `mirage` tool used in the `ci.sh` script... another solution may be to have a travis-mirage (but i'm a bit too lazy to put any effort into that, or look whether it exists and works) -- this is basically the same as mirage/mirage-skeleton and roburio/unikernels repositories are working